apps/tangle-website/src/components/Socials.tsx

```diff
@@ -0,0 +1,60 @@
+import { IconBase } from '@webb-tools/icons/types';
```

Comment:
@AtelyPham thoughts on having this live in the webb-ui-kit?

Reply:
We already have this component in the Footer component of the webb-ui-kit (you can check it out [here](https://github.com/webb-tools/webb-dapp/blob/develop/libs/webb-ui-components/src/components/Footer/Footer.tsx#L51-L62)). You can extract it into a separate component and utilize it here. @vutuanlinh2k2

Reply:
Thanks for your review @dutterbutter @AtelyPham. I will create a separated component for Socials in webb-ui-kit